enterprise-cfengine-guide/settings.markdown

```diff
@@ -196,6 +196,33 @@ Mission portal can authenticate against an external directory.
 
 **See Also:** [LDAP authentication REST API][LDAP authentication API]
 
+
+## Role based access control ##
+
+![Role based access control](role_based_access_control_settings.png)
+
+Roles in Mission portal can be restricted to perform only configured actions. 
+And you can configure they on Role based access control settings page.
+
+**Special Notes:** 
+
+- Admin role has all permissions by default.
+
+- Cf_remoteagent role has all permissions related to API by default.
+
+- User with several roles will be permitted to perform actions from every role.
```

Comment:
I would reword this slightly:

Permissions granted by roles are additive, users with multiple roles are permitted to perform actions granted by each role.